server/services/slack/index.js

```diff
@@ -1,7 +1,13 @@
 // @flow
+import type { Event } from '../../events';
+
 const Slack = {
   id: 'slack',
   name: 'Slack',
+  on: (event: Event) => {
+    console.log(`Slack service received ${event.name}`);
```

Comment:
Example implementation, this would likely end up a switch statement or an `if` filter depending on the complexity of the integration and how many types of events it's looking to handle.